Source/Scene/GltfLoader.js

```diff
@@ -69,6 +69,7 @@ var GltfLoaderState = {
  * @param {Boolean} [options.incrementallyLoadTextures=true] Determine if textures may continue to stream in after the glTF is loaded.
  * @param {Axis} [options.upAxis=Axis.Y] The up-axis of the glTF model.
  * @param {Axis} [options.forwardAxis=Axis.Z] The forward-axis of the glTF model.
+ * @param {Boolean} [options.loadAsTypedArray=false] Load all attributes as typed arrays instead of GPU buffers.
```

Comment:
> Load all **attributes** as typed arrays instead of GPU buffers.

The indices are also loaded as typed arrays. Does that need to be explicitly mentioned here? Not sure if the `attributes` umbrella implicitly covers those too.